migrations/2_deploy_libraries.js

```diff
@@ -1,11 +1,7 @@
 const SortedDoublyLL = artifacts.require("SortedDoublyLL")
 const BondingManager = artifacts.require("BondingManager")
 
-module.exports = function(deployer, network) {
-    if (network === "unitTest") {
```

Comment:
Removing this check would result in the deployment of `SortedDoublyLL` before each unit test is run (i.e. when the `unitTest` network is used).

Could an alternative be to only deploy and link `SortedDoublyLL` in `test/unit/BondingManager.js` (and generally in any unit test that needs to deploy the BondingManager)?

i.e. In `test/unit/BondingManager.js`

```
before(async () => {
    ...
    const ll = await SortedDoublyLL.new()
    // According to the truffle-contract docs https://github.com/trufflesuite/truffle/tree/develop/packages/contract#mycontractlinkinstance
    // BondingManager.link(ll) should also work, but I encountered an error when I tried
    // Worth double checking if the above works as it would be a bit cleaner since we wouldn't need to also specify the library name
    // Otherwise, the below should be fine
    BondingManager.link("SortedDoublyLL", ll.address)
})
```

Reply:
fixed in 96c17e6